resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Config.scala

```diff
@@ -684,6 +684,15 @@ private[spark] object Config extends Logging {
       .checkValue(value => value > 0, "Maximum number of pending pods should be a positive integer")
       .createWithDefault(Int.MaxValue)
 
+  private [spark] val KUBERNETES_EXECUTOR_POD_ANTI_AFFINITY =
+    ConfigBuilder("spark.kubernetes.executor.pod.antiAffinity.enable")
+      .doc("If enable, register executor with anti affinity. This anti affinity will help " +
+        "Kubernetes assign executors of the same Application to different nodes " +
+        "as much as possible")
+      .version("3.2.1")
+      .booleanConf
+      .createWithDefault(false)
```

Comment:
The default value is correct because AntiAffinity could hurt EKS AutoScale feature.